src/libfuncs/int.rs

```diff
@@ -636,7 +637,7 @@ fn build_square_root<'ctx, 'this>(
             CoreTypeConcrete::Uint32(_) => (32, 16),
             CoreTypeConcrete::Uint64(_) => (64, 32),
             CoreTypeConcrete::Uint128(_) => (128, 64),
-            _ => unreachable!(),
+            _ => native_panic!("unhandled type in int square root"),
```

Comment:
Rather than unhandled it's invalid because there are no more integers.
An example: it doesn't make sense to compute the integer square root of a struct.
```suggestion
            _ => native_panic!("invalid value type in int square root"),
```